### PR TITLE
Kubernetes blog post

### PR DESCRIPTION
**Pull request by whiskerwind**
Adding Kubernetes blog post

### CODE REVIEW

File: blogs/Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h1>Kubernetes</h1>
+
+<p>Kubenetes is an open source system originally developed by google to aid developers in the automation of deploying and managing applications inside containers across multiple machines.
+</p>
+<img src="https://octoperf.com/img/blog/kraken-kubernetes-ingress-nginx-frontend/kubernetes.png" width="400">
+<caption>kubenetes logo</caption>
```
Comment (NishkarshRaj) [inline blogs/Kubernetes.md]:
```suggestion
<p align="center">
<img src="https://octoperf.com/img/blog/kraken-kubernetes-ingress-nginx-frontend/kubernetes.png" width="400">
</p>
```

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
@whiskerwind make following changes:

1. Center align the image attached - already suggested in comments
2. Add blog link in this PR

**Comment by whiskerwind** [comment]
thanks for the feedback 👍  I have updated the PR

**Review by NishkarshRaj — APPROVED** [review]
Thanks @whiskerwind for your contribution 👍

**Pull request merged**